File: Assignment_12/Program1.c
//Accept the number of rows and number of columns from the user and display the below patter

// * * * *
// * * * *
// * * * *
// * * * *


#include<stdio.h> 
void Pattern(int iRow , int iCol)
{
    int i = 0, j = 0;

    for(i = 1 ; i < iRow ; i++)
    {
        for(j = 0 ; j < iCol ; j++)
        {
            printf(" * ");
        }
        printf("\n");
    }
}
int main()
{
    int iValue1 = 0;

    printf("Enter the number of rows\n");

    scanf("%d",&iValue1);

    int iValue2 = 0;

    printf("Enter the number of columns\n");

    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Assignment_15/Program1.java
/*
*   Accept the number of rows and columns from user and print the below pattern 
*   * # # #
*   * * # #
*   * * * #
*   * * * *
*/

import java.util.*;

class Demo
{
    public  int iRow ;
    public  int iCol;
    public int i ;
    public int j;

    public Demo(int A, int B)
    {
        this.iRow = A;
        this.iCol = B;
        this.i = 0;
        this.j = 0;
    }


    public void Pattern()
    {
        for(i = 1 ; i<= iRow ;i++)
        {
            for(j = 1; j <= iCol ; j++)
            {
                if(i > j || i == j)
                {
                    System.out.print("* ");
                }
                else 
                {
                    System.out.print("# ");
                }
            }
            System.out.println();
        }
    }
}

class Program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of rows");

        int iValue1 = sobj.nextInt();

        System.out.println("Enter the numbr of columns");

        int iValue2 = sobj.nextInt();

        Demo dobj = new Demo(iValue1,iValue2);

        dobj.Pattern();

        // Pattern(iValue1,iValue2);
    }
}